lib/encoding/crypto.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Elixium.Crypto do
```

Comment:
Most of the functions in this module belong in the Elixium.Utilities module. (Thats currently where the merkle tree stuff and the sha stuff is)

Reply:
Shifted the necessary ones into the Utilities module